src/video/x11/SDL_x11keyboard.c

```diff
@@ -841,6 +842,38 @@ X11_SetTextInputRect(_THIS, const SDL_Rect *rect)
 #endif
 }
 
+SDL_bool
+X11_HasScreenKeyboardSupport(_THIS)
+{
+    SDL_VideoData *videodata = (SDL_VideoData *) _this->driverdata;
+    return videodata->is_steam_deck;
+}
+
+void
+X11_ShowScreenKeyboard(_THIS, SDL_Window *window)
+{
+    SDL_VideoData *videodata = (SDL_VideoData *) _this->driverdata;
+
+    if (videodata->is_steam_deck) {
+        SDL_OpenURL("steam://open/keyboard?XPosition=0&YPosition=0&Width=0&Height=0&Mode=0");
```

Comment:
0 is the default value for the position and size parameters, so you can omit those.

The mode can be one of:
```c
enum EFloatingGamepadTextInputMode
{
	k_EFloatingGamepadTextInputModeModeSingleLine = 0,		// Enter dismisses the keyboard
	k_EFloatingGamepadTextInputModeModeMultipleLines = 1,	// User needs to explictly close the keyboard
	k_EFloatingGamepadTextInputModeModeEmail = 2,			// Keyboard layout is email, enter dismisses the keyboard
	k_EFloatingGamepadTextInputModeModeNumeric = 3,			// Keyboard layout is numeric, enter dismisses the keyboard
};
```

Reply:
Maybe we can add a hint for the keyboard mode and implement that for the other onscreen keyboards as well? (as a separate PR?)

Reply:
The default mode for steam://open/keyboard is `k_EFloatingGamepadTextInputModeModeMultipleLines`

Reply:
I'll change the mode to 0 - for the rest I'm leaving them in just so we have it documented _somewhere_ - I'm also adding a link to the SteamUtils documentation just to be 100% sure this URL is legible to others who find this!

(The hint idea sounds good too, we should consider this for a 3.0 API maybe?)